NitroxServer/ConsoleCommands/ExitCommand.cs

```diff
@@ -7,11 +8,13 @@ public class ExitCommand : Command
         public ExitCommand()
         {
             Name = "exit";
+            Alias = new string[] { "stop", "halt" };
         }
 
         public override void RunCommand(string[] args)
         {
             Server.Instance.Stop();
+            //NitroxServiceLocator.EndCurrentLifetimeScope();
```

Comment:
Something left over?

Reply:
Tried using it to end the lifetime scope but server didnt like it